File: PHP Ejercicio Estructurado/totalventas.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Total de Ventas </title>
</head>
<body>
    

<form action="totalventas.php" method="post">
<p> Digite el código del Producto: </p>
<input type="text" name="codigo" id="" placeholder="Digite el código del Producto:"> 

<p> Digite el nombre del Producto: </p>
<input type="text" name="nombre" id="" placeholder="Digite el nombre del Producto:"> 

<p> Digite el Precio del Producto:  </p>
<input type="number" name="precio" id="" placeholder="Digite el Precio del Producto: "> 
    

<p> Digite la Cantidad del Producto:  </p>
<input type="number" name="cantidad" id="" placeholder="Digite la Cantidad del Producto:"> 
        
<p> Digite las unidades Vendidas del Producto: </p>
<input type="number" name="unidades" id="" placeholder="Digite las unidades Vendidas del Producto:"> 

<input type="hidden" name="NumeroProductos" value="<?php echo $NumeroProductos?>">
    
<input type="submit" value="Registrar Productos">


</form>
    
    <?php

$TotalProductos = 0;
  

if(isset($_POST["codigo"]) && isset($_POST["nombre"]) && isset($_POST["precio"]) 
&& isset($_POST["cantidad"]) && isset($_POST["unidades"])  ){

    echo "<p> Existe valores de los productos   </p>";


   
   $codigo =  $_POST["codigo"] ;
   $nombre =  $_POST["nombre"] ;
   $precio =  $_POST["precio"] ;
   $cantidad =  $_POST["cantidad"] ;
   $unidades =  $_POST["unidades"] ;

  
    
    $TotalProducto = $precio * $unidades;

    $Diferencia = $TotalProducto - $TotalProducto ;

    $TotalCantidad = $cantidad - $unidades;

echo "El Total del Producto ".$nombre." es: $".$TotalProducto." COP"."<br>".
"El Balance es de: $Diferencia ".", con N° de Productos es de: $TotalCantidad";

$TotalProductos += $TotalProducto;
  

}else {

  echo "<p> No existe valores de los productos   </p>" ; 
}

  

  ?>











</body>
</html>

File: Ejercicio Base/Código Java/Total_Ventas2/Total_Ventas.java
import java.util.Scanner;

public class Total_Ventas {

    public static void main(String[] args) {
		
		Scanner Entrada = new Scanner(System.in);
		
		System.out.println("Digite la Cantidad de Productos que desea registrar: ");
		int Productos_Registrados = Entrada.nextInt();
		
		int TotalProductos=0;
		
		for(int i=1;i<=Productos_Registrados;i++){
		    
		    System.out.println("Digite el código del Producto: ");
		String Codigo_Producto = Entrada.next();
		
		   System.out.println("Digite el nombre del Producto: ");
		String Nombre_Producto = Entrada.next();
		
		   System.out.println("Digite el Precio del Producto: ");
		  int Precio_Producto = Entrada.nextInt();
		    
		     System.out.println("Digite la Cantidad del Producto: ");
		int Cantidad_Producto = Entrada.nextInt();
		
		   System.out.println("Digite las unidades Vendidas del Producto: ");
		int Unidades_Vendidas = Entrada.nextInt();
		
		int TotalProducto = Precio_Producto * Unidades_Vendidas;
		
		System.out.println("El Total de Ventas del "+ Nombre_Producto+ " es: " + TotalProducto);
		
		
		 TotalProductos+=TotalProducto;
		
		    
		}
		
		System.out.println("El Total de Ventas de Todos los Productos es: " + TotalProductos);
		
	}

    
}
